src/styles/mixins/layout.scss

```diff
@@ -76,7 +76,7 @@
  * @FIXME Find workaround for SvgIcon exception
  */
 @mixin vertical-rhythm($size: var.$layout-space) {
-  & > * + *:not([class*='svg-icon--inline']) {
+  & > * + *:not([class*='SvgIcon--inline']) {
```

Comment:
This coupling is now not very ideal since its in a sub-package, but we can address that later... its also possible this may move into the shared package if this is general enough and useful in LMS.